src/list.json

```diff
@@ -120,7 +120,18 @@
     ],
     "N": [],
     "O": [],
-    "P": [],
+    "P": [
+      {
+               "organization": "Percona",
```

Comment:
And after that run this for Re-styling
git checkout <your branch>
curl https://restyled.io/gh/monizb/repos/hacktoberfest-swag-list/jobs/2332898/patch | git am
git push